flycheck.el

```diff
@@ -232,6 +232,7 @@ attention to case differences."
     rust-cargo
     rust
     sass
+    sass-lint
```

Comment:
Shouldn't we use `sass` before `sass-lint`?  `sass` is a very basic checker that only catches some syntax errors; shouldn't we generally prefer `sass-lint` if it's available?